From: Robert Luberda <robert@debian.org>
Date: Sun, 25 Oct 2015 15:37:28 +0100
Subject: 0034 Fix munchlist failure

Ignore failure of egrep command that creates $CROSSROOTS,
because it might happen that $LEGALLIST does not have
any root with multiple cross-product flags.

This fixes bug introduced in 0032-Check-munchlist-errors.patch.

Bug-Debian: https://bugs.debian.org/794152
---
 munchlist.X | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/munchlist.X b/munchlist.X
index d4082a6..2e98a39 100755
--- a/munchlist.X
+++ b/munchlist.X
@@ -712,7 +712,7 @@ then
 '`
     egrep "$flagmarker.*[$preflags].*[$sufflags]|$flagmarker.*[$sufflags].*[$preflags]" \
       $LEGALFLAGLIST \
-      > $CROSSROOTS
+      > $CROSSROOTS || :
 
     #
     # We will need an awk script;  it's so big that it core-dumps my shell
